src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/PemEncoding.cs

```diff
@@ -168,9 +168,9 @@ static ReadOnlySpan<char> WritePostEB(ReadOnlySpan<char> label, Span<char> desti
             {
                 int size = PostEBPrefix.Length + label.Length + Ending.Length;
                 Debug.Assert(destination.Length >= size);
-                PostEBPrefix.AsSpan().CopyTo(destination);
+                PostEBPrefix.CopyTo(destination);
                 label.CopyTo(destination.Slice(PostEBPrefix.Length));
-                Ending.AsSpan().CopyTo(destination.Slice(PostEBPrefix.Length + label.Length));
+                Ending.CopyTo(destination.Slice(PostEBPrefix.Length + label.Length));
```

Comment:
Looks like this is a netcoreapp2.1 assembly? Surprised CI isn't complaining.

Reply:
I don't think so?
https://github.com/dotnet/runtime/blob/edce31f285be386c952a8f538be1b5e3642fe5c1/src/libraries/System.Security.Cryptography.Encoding/src/System.Security.Cryptography.Encoding.csproj#L4

Reply:
Well, then it looks like I should stop paying attention to source.dot.net. :)

https://source.dot.net/System.Security.Cryptography.Encoding/System.Security.Cryptography.Encoding.csproj.html

```xml
<TargetFramework>netcoreapp2.1</TargetFramework>
```

Reply:
@alexperovich, where is that file coming from?  I can't find anything in the repo or in the history of the csproj that looks like that?